File: Lua/1012.lua
a, b, c = io.read('*n', '*n', '*n')
pi = 3.14159

aTri = (a*c) / 2
aC = pi * math.pow(c, 2)
aT = ((a + b) * c) / 2
aQ = math.pow(b, 2)
aRet = a * b

aTri = string.format( "TRIANGULO: %.3f", aTri)
aC = string.format( "CIRCULO: %.3f", aC)
aT = string.format( "TRAPEZIO: %.3f", aT)
aQ = string.format( "QUADRADO: %.3f", aQ)
aRet = string.format( "RETANGULO: %.3f", aRet)

print(aTri)
print(aC)
print(aT)
print(aQ)
print(aRet)

File: C/1012.c
#include <stdio.h>

int main(){
    double A, B, C;
    double pi = 3.14159;
    float tri, cir, tra, qua, ret;
    scanf("%lf %lf %lf", &A, &B, &C);
    tri = (A * C)/2;
    cir = pi * pow(C, 2);
    tra = ((A + B) * C ) / 2;
    qua = pow(B, 2);
    ret = A * B;
    printf("TRIANGULO: %.3f\n", tri);
    printf("CIRCULO: %.3f\n", cir);
    printf("TRAPEZIO: %.3f\n", tra);
    printf("QUADRADO: %.3f\n", qua);
    printf("RETANGULO: %.3f\n", ret);
    return 0;

}